src/index.ts

```diff
@@ -151,4 +169,9 @@ class DahuaMotionAlertsPlatform implements IndependentPlatformPlugin {
 		
 		return error
 	}
+}
+
+type DahuaEventsConfig = {
+    events:       Set<string>

```

Comment:
Can you just add this to the `CameraConfig` struct instead of creating a new struct?

Reply:
This is grouping per host, not by camera. Cannot match full struct contract and populate index and cameraName